### PR TITLE
Update TSLA transform with description, date, creator, and spatial.

### PR DESCRIPTION
**Pull request by mlhale7**
## What's new?

I added the updated transform for TSLAqdctomods and removed the old transform outside of the XSLT folder.

* Modified transform to do include description, date, creator, and spatial tags for TSLA data.

## How should this be tested?

A description of what steps someone could take to:

* Does the transform validate in Oxygen using th Saxon 8.7 processor
* Using sample XML code, does the pull request doe what is intended
* If you have access to a Repox instance, does applying the pull request and touching all existing XSLT do what is intended.
* If you run a full Scema validation on the feed the transform(s) applies to(using Variety.js and DLTN Metadata QA), does every record have:
	* a titleInfo/title
	* a location/url
	* an accessCondition

## Interested parties

@CanOfBees @markpbaggett

### CODE REVIEW

File: XSLT/tslaqdctomods.xsl [hunk 1]
```diff
@@ -58,14 +58,26 @@
...
+          <!-- description -->
+          <xsl:apply-templates select="dc:description"/>
+          <!-- date -->
+          <xsl:apply-templates select="dc:date"/>
+          <!-- spatial -->
+          <xsl:apply-templates select="dcterms:spatial"/>
+          <recordInfo>
+              <recordContentSource>Tennessee State Library &amp; Archives</recordContentSource>
+              <languageOfCataloging>
+                  <languageTerm term="code" authority="iso639-2b">eng</languageTerm>
```
Comment (markpbaggett) [inline XSLT/tslaqdctomods.xsl]: This isn't valid.  Should be:
`
<languageTerm type="code" authority="iso639-2b">eng</languageTerm>`

File: XSLT/tslaqdctomods.xsl [hunk 2]
```diff
@@ -80,5 +92,33 @@
...
+        </xsl:for-each>
+    </xsl:template>
+    
+    <!-- description -->
+    <xsl:template match="dc:description">
+        <abstract><xsl:apply-templates/></abstract>
+    </xsl:template>
+    
+    <!-- date -->
+    <xsl:template match="dc:date">
```
Comment (markpbaggett) [inline XSLT/tslaqdctomods.xsl]: Are we just worried about edm:TimeSpan/dc:date (not edm:TimeSpan/edm:begin)?  If so, this is good.

File: XSLT/tslaqdctomods.xsl (hunk 1 above)
Comment (mlhale7) [reply XSLT/tslaqdctomods.xsl]: Thanks for catching this - sorry for the copy and paste error.

File: XSLT/tslaqdctomods.xsl (hunk 2 above)
Comment (mlhale7) [reply XSLT/tslaqdctomods.xsl]: That's all I was initially worried about, but should we consider adding the edm:TimeSpan/edm:begin in the future?

**Pull request merged**